dwc_server.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+import os
+#import asyncio
+from fastapi import FastAPI, File, UploadFile
+from fastapi.responses import FileResponse
+#import websockets
+import copy
+import numpy as np
+import uuid
+
+app = FastAPI()
+
+datasets = []
+num_clients = 1
+model = None
+priors = []
+client_ids = []
+
+@app.get("/")
+async def root():
+    return "root"
+
+@app.get("/{model}/")
+async def update_model(model: tf.keras.Model):
+    return FileResponse('consolidated.h5')
```

Comment:
this should return the consolidated model that's saved in the code below. i don't think the filename is as shown here.